src/server/electron/electron.ts

```diff
@@ -87,7 +87,7 @@ export class ElectronApplication extends SdkObject {
     this._windows.add(page);
 
     // Below is async.
-    const handle = await this._nodeElectronHandle!.evaluateHandle(({ BrowserWindow }, windowId) => BrowserWindow.fromId(windowId), windowId).catch(e => {});
+    const handle = await (await this._nodeElectronHandle)!.evaluateHandle(({ BrowserWindow }, windowId) => BrowserWindow.fromId(windowId), windowId).catch(e => {});
```

Comment:
In case of mentioned race, this._nodeElectronHandle can still be undefined here.